utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/dnsmasq.lua

```diff
@@ -0,0 +1,41 @@
+local ubus = require "ubus"
+
+local function scrape_leasefile(name, leasefile)
+  metric_dhcp_lease = metric("dhcp_lease", "gauge")
+  for e in io.lines(leasefile) do
+    local fields = space_split(e)
+    if fields[4] ~= nil then
+      local labels = {
+	dnsmasq = name,
```

Comment:
name is not super interesting, I think it even changes on each reboot 
maybe put the lease file ?